### DISCUSSION TITLE
How to get yaml.min.js?

### DISCUSSION THREAD

**Discussion by WTIFS** [eemeli/yaml#589] [Q&A] [answered]
Hi there, I want to use this library in browser. 
Is there any way I can generate and get a yaml.min.js file?

**Comment by eemeli** [comment, marked as answer]
By bundling and minifying it, the same as with any other library? Something like this should work:
```
npm install --no-save yaml rollup @rollup/plugin-terser
npx rollup ./node_modules/yaml/browser/dist/index.js --plugin terser --file yaml.min.js --format umd --name YAML
```

**Reply by WTIFS** [reply]
Thanks a lot! I'm new to node.js and this works
